src/parser/DASHTree.cpp

```diff
@@ -1304,6 +1313,15 @@ end(void *data, const char *el)
         dash->currentNode_ &= ~MPDNODE_BASEURL;
       }
     }
+    else if (strcmp(el, "Location") == 0)
```

Comment:
Also this part should be checked identically as other parts, first check for set bit, and then for the string

Reply:
Okay, I think I fixed this in https://github.com/peak3d/inputstream.adaptive/pull/303